### DISCUSSION TITLE
Sidekiq Logging

### DISCUSSION THREAD

**Discussion by reidmorrison** [reidmorrison/rails_semantic_logger#228] [General]
Discuss any specific issues, or ideas for improving the Sidekiq logging.

For example:
- Plain text log messages that should be converted into structured logging?
- Missing context information in logs?
- Ideas on additional logging that could be added?
- Discussions around the metrics that were added, or others we could add? (`sidekiq.queue.latency`,`sidekiq.job.perform`)

**Comment by jdelStrother** [comment]
I think it's pretty good as is. I do still rely on some custom middleware that logs the arguments to `perform`, which naively sounds like it would be an easy addition. I don't know if it would be a sensible default for rails_semantic_logger though - our version is little coupled into the Job classes, though, because we whitelist the parameters that can be logged to avoid logging sensitive data.
